src/test/java/reposense/commits/CommitInfoAnalyzerTest.java

```diff
@@ -118,4 +123,62 @@ public void analyzeCommits_emailWithAdditionOperator_success() {
 
         Assert.assertEquals(NUMBER_MINGYI_COMMIT, commitResults.size());
     }
+
+    @Test
+    public void analyzeCommits_multipleCommitsWithCommitMessageBody_success() throws ParseException {
+        Author author = new Author(JINYAO_AUTHOR_NAME);
+        List<CommitResult> expectedCommitResults = new ArrayList<>();
+        expectedCommitResults.add(new CommitResult(author, "2eccc111e813e8b2977719b5959e32b674c56afe",
+                parseGitStrictIsoDate("2019-06-19T13:02:01+08:00"), ">>>COMMIT INFO<<<",
+                "Hi there!\n\n>>>COMMIT INFO<<<\n", 1, 0));
+        expectedCommitResults.add(new CommitResult(author, "8f8359649361f6736c31b87d499a4264f6cf7ed7",
+                parseGitStrictIsoDate("2019-06-19T13:03:39+08:00"), "[#123] Reverted 1st commit",
+                "This is a test to see if the commit message body works. "
+                + "All should be same same.\n>>>COMMIT INFO<<<\n|The end.|\n", 0, 1));
+
+        config.setBranch("751-CommitInfoAnalyzerTest-analyzeCommits_multipleCommitsWithCommitMessageBody_success");
+        config.setAuthorList(Collections.singletonList(author));
+        config.setSinceDate(new GregorianCalendar(2019, Calendar.JUNE, 19).getTime());
+        config.setUntilDate(new GregorianCalendar(2019, Calendar.JUNE, 20).getTime());
+
+        List<CommitInfo> actualCommitInfos = CommitInfoExtractor.extractCommitInfos(config);
+        List<CommitResult> actualCommitResults = CommitInfoAnalyzer.analyzeCommits(actualCommitInfos, config);
+
+        Assert.assertEquals(expectedCommitResults, actualCommitResults);
+    }
+
+    @Test
+    public void analyzeCommits_commitsWithEmptyCommitMessageTitleOrBody_success() throws ParseException {
+        Author author = new Author(JINYAO_AUTHOR_NAME);
+        List<CommitResult> expectedCommitResults = new ArrayList<>();
+        // 1st test: Contains commit message title but no commit message body.
+        expectedCommitResults.add(new CommitResult(author, "aa7044dd2fa7b5244bdcfd27da09e34ed7d3ef1d",
+                parseGitStrictIsoDate("2019-07-01T12:18:56+08:00"), "Test 1: With title but no body",
+                "", 1, 0));
+        // 2nd test: Contains no commit message title and no commit message body.
+        expectedCommitResults.add(new CommitResult(author, "db5f49e2ed5f72090f1b7fd66920661bfcaf0bae",
+                parseGitStrictIsoDate("2019-07-01T12:21:56+08:00"), "",
+                "", 0, 1));
+        // 3rd test: Contains both commit message title and commit message body.
```

Comment:
No need for the third as it does not match the title (intention) of this test. This case should be covered in the previous one right?

Reply:
Yes covered in the previous case. 

Removed the third test case